doc/psidk/src/development/services/rust-service/package.md

```diff
@@ -7,6 +7,7 @@ A [psibase package](../../../specifications/data-formats/package.md) may be comp
 The available fields are:
 
 - `package-name`: Must be present on the top-level crate for the package.
+- `description`: A description of the package.
```

Comment:
`description` and `version` are taken from `[package]`, not `[package.metadata.psibase]`

Reply:
Oh. I'll remove this and also move the description field in the `test_package` to `[package]`.

Reply:
Done